### ISSUE TITLE
Analyze with SonarCloud

### ISSUE THREAD

**Issue by keilw** [unitsofmeasurement/unit-api#48]
See https://about.sonarcloud.io/news/2016/09/01/sonarqubedotcom-open-to-aynone.html Either via Maven or CircleCI (if possible similar to Travis examples) the project should be analyzed.

**Comment by Oliver-Loeffler** [comment]
Technically analysis works well when following Circle-CI is modified accordinly:

```yml
build:
    docker:
    environment:
    steps:
     # share coverage
     # perform sonar analysis
    - run: mvn sonar:sonar -Dsonar.organization=MY_ORGANIZATION -Dsonar.host.url=MY_HOST    -Dsonar.login=MY_LOGIN
```

The values for `MY_ORGANIZATION`, `MY_LOGIN` and `MY_HOST` can be configured usinh Circle-CI environment variables probably. But, I'm not sure right now how it works with encryption so that the login token stays secret.

I've played offline and 'online' with it and it turned out, that the token "javax.measure:unit-api" is already occupied. For sonarcloud.io it is mandatory that the token is uniqe whereas the token is usually groupId:artifactId for a Maven project.

So following token exists w/o content: 
https://sonarcloud.io/dashboard?id=javax.measure:unit-api 

However, when datas is e.g. uploaded to:
https://sonarcloud.io/dashboard?id=net.raumzeitfalle.javax.measure:unit-api 
the analysis works fine. No need to change POM.xml etc.

Who owns the already existing token? However, I'm going to figure out to customize the project token.

**Comment by Oliver-Loeffler** [comment]
So regardless, which organization is behind, a custom project key can be defined using:

```yml
    - run: mvn sonar:sonar -Dsonar.organization=MY_ORGANIZATION \
                                        -Dsonar.projectKey=MY_PROJECT_KEY \
                                        -Dsonar.host.url=MY_HOST  \
                                        -Dsonar.login=MY_LOGIN
```

Therefore, when there is a project related organisation or entity in sonarcloud.io, the unit-api analysis can be stored using a customized project key.

**Comment by keilw** [comment]
Fixed by #111
